src/main/java/com/openmrs/migrator/core/services/impl/DataBaseServiceImpl.java

```diff
@@ -14,19 +21,25 @@
 
   private final ConfigurationStore configurationStore;
 
+  private final FileIOUtilities fileIOUtilities;
+
   @Autowired
-  public DataBaseServiceImpl(CommandService commandService, ConfigurationStore configurationStore) {
+  public DataBaseServiceImpl(
+      CommandService commandService,
+      ConfigurationStore configurationStore,
+      FileIOUtilities fileIOUtilities) {
     this.commandService = commandService;
     this.configurationStore = configurationStore;
+    this.fileIOUtilities = fileIOUtilities;
   }
 
   @Override
-  public void importDatabaseFile(String databaseName, String fileName) {
+  public void importDatabaseFile(String databaseName, String fileName) throws IOException {
     commandService.runCommand(
         "mysql",
-        "-u" + configurationStore.getDatabaseUser(),
-        "-p" + configurationStore.getDatabasePassword(),
-        "-h" + configurationStore.getDatabasePassword(),
+        "-u" + fileIOUtilities.getValueFromConfig("username", "="),
```

Comment:
please change this back. We want all config to be stored in the settingsService so that we are all reading the config from one source.

We should be using the work done in ticket 100